.github/workflows/ci.yml

```diff
@@ -270,6 +270,11 @@ jobs:
       - run: cargo install sqlx-cli
       - run: bash scripts/run_migrations.bash
 
+      - name: Install wasm-snip
+        run: cargo install wasm-snip
+      - name: Build test WASM blobs
+        run: bash scripts/utils/build_test_wasm_module.bash
```

Comment:
`build_test_wasm_module` builds 4 indexers but we only need 2 for testing (`fuel-indexer-test` and `simple-wasm`) so maybe we can just build those?